src/utils/string/StringUtils.js

```diff
@@ -3,6 +3,7 @@
 import * as React from 'react';
 import moment from 'moment';
 import reactStringReplace from 'react-string-replace';
+import _, {isArray} from 'lodash';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for lodash is inconsistent. It's better to stick with one style of importing for consistency and readability. Here, you're importing the entire lodash library as `_` and also importing `isArray` separately from lodash. You should only import what you need.
```diff
- import _, {isArray} from 'lodash';
+ import { isArray } from 'lodash';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->